index.html

```diff
@@ -36,13 +34,160 @@ <h2>
+         <ul id="lang-skills">
+            <li id="css">css</li>
+            <li id="html">html</li>
+            <li id="bootstrap">bootstrap</li>
+            <li id="Ruby">Ruby</li>
+       </ul>
         
-      </section>
+       <button id="project-btn">
+            See Project
+    </button>
+</div>
+    </section> 
+
+    <section id="card">
+        <div id="placehokder-img-div">
+            <div class="placehokder-img">
+        <h1 id="placeholder-img-head">Profesional Art Printing Data</h1>
+        <p id="palceholder-img-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="lang-skills">
+            <li class="html">html</li>
+            <li class="bootstrap">bootstrap</li>
+            <li class="Ruby">Ruby</li>
+       </ul>
+       <button id="project-btn">See Project
+       </button>
+    </div>
+    </div>
+    <div id="placehokder-img-div">
+        <div class="placehokder-img">
+    <h1 id="placeholder-img-head">Profesional Art Printing Data</h1>
+    <p id="palceholder-img-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+    <ul class="lang-skills">
+        <li class="html">html</li>
+        <li class="bootstrap">bootstrap</li>
+        <li class="Ruby">Ruby</li>
+   </ul>
+   <button id="project-btn">See Project
+   </button>
+</div>
+</div>
+<div id="placehokder-img-div">
+    <div class="placehokder-img">
+<h1 id="placeholder-img-head">Profesional Art Printing Data</h1>
+<p id="palceholder-img-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+<ul class="lang-skills">
+    <li class="html">html</li>
+    <li class="bootstrap">bootstrap</li>
+    <li class="Ruby">Ruby</li>
+</ul>
+<button id="project-btn">See Project
+</button>
+</div>
+</div>
+<div id="placehokder-img-div">
+    <div class="placehokder-img">
+<h1 id="placeholder-img-head">Profesional Art Printing Data</h1>
+<p id="palceholder-img-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+<ul class="lang-skills">
+    <li class="html">html</li>
+    <li class="bootstrap">bootstrap</li>
+    <li class="Ruby">Ruby</li>
+</ul>
+<button id="project-btn">See Project
+</button>
+</div>
+</div>
+<div id="placehokder-img-div">
+    <div class="placehokder-img">
+<h1 id="placeholder-img-head">Profesional Art Printing Data</h1>
+<p id="palceholder-img-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+<ul class="lang-skills">
+    <li class="html">html</li>
+    <li class="bootstrap">bootstrap</li>
+    <li class="Ruby">Ruby</li>
+</ul>
+<button id="project-btn">See Project

```

Comment:
- [ ] kindly update your card's button styling to match the Figma template , please update all of the buttons of your cards 🙏 

Your Design           | Figma
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/198849761-f3516f07-659b-43db-befb-7bb28f520342.png) | ![image](https://user-images.githubusercontent.com/89282221/198849784-01bee9c7-e8b5-4b2f-80e8-02f2558b2d79.png)